elements/userlevel/fromdpdkdevice.cc

```diff
@@ -405,7 +405,7 @@ void FromDPDKDevice::selected(int fd, int mask) {
 
 enum {
     h_vendor, h_driver, h_carrier, h_duplex, h_autoneg, h_speed, h_type,
-    h_ipackets, h_ibytes, h_imissed, h_ierrors, h_nombufs,
+    h_ipackets, h_ibytes, h_imissed, h_ierrors, h_nombufs, h_out_of_buffer,
```

Comment:
This is already possible with xstats I think, where you can give a name. "fd.xstats out_of_buffer", no? Because we could do this for all xstats.

Reply:
Ops, I missed that. I added an indication of that to the documentation.